module/VuFind/tests/unit-tests/src/VuFindTest/Mailer/MailerTest.php

```diff
@@ -277,7 +277,7 @@ public function testSendWithSubjectNotAllowedInBody()
 
                 body
                 EOT,
-            subjectInBody: false
+            subjectInBody:false
```

Comment:
Reverting a minor whitespace issue:

```suggestion
            subjectInBody: false
```